src/routes/embedded/account/backup-wallet/backup-wallet-qrcode.tsx

```diff
@@ -0,0 +1,102 @@
+import type { JWKInterface } from "@dha-team/arbundles";
+import { useState, useCallback, useEffect } from "react";
+import { toast } from "react-toastify";
+import { Flex } from "~components/common/Flex";
+import { CopyToClipboard } from "~components/CopyToClipboard";
+import { Text } from "~components/embed/ui";
+import { OnboardingCard } from "~components/embed/ui/molecules/card/onboarding-card/OnboardingCard";
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
+import { useLocation } from "~wallets/router/router.utils";
+import type { LocalWallet } from "~wallets/wallets.types";
+import browser from "webextension-polyfill";
+import { dataToFrames } from "qrloop";
+import { freeDecryptedWallet } from "~wallets/encryption";
+import { QRCodeWrapper } from "~components/QRCodeWrapper";
+import { QRCodeLoop } from "~components/QRCodeLoop";
+import { useTheme } from "~components/embed/contexts/ThemeContext";
+
+export function AccountBackupWalletQrCodeEmbeddedView() {
+  const { isDarkMode } = useTheme();
+  const { navigate } = useLocation();
+  const { getDecryptedWallet } = useEmbedded();
+  const [isLoading, setIsLoading] = useState(false);
+  const [copied, setCopied] = useState(false);
+  const [frames, setFrames] = useState<string[]>([]);
```

Comment:
I think it's better to use an object with number keys in here (`Record<number, string>`), as we should use a method similar to `freeDecryptedWallet()` for it.